api/relations/v0/lookup.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package kessel.relations.v0;
+
+import "google/api/annotations.proto";
+import "relations/v0/common.proto";
+
+option go_package = "ciam-rebac/api/relations/v0;v0";
+option java_multiple_files = true;
+option java_package = "org.project_kessel.api.relations.v0";
+
+service KesselLookupService {
+    rpc LookupSubjects(LookupSubjectsRequest) returns (stream LookupSubjectsResponse) {
+        option (google.api.http) = {
+            get: "/v1/subjects"
+        };
+    };
+}
+
+message LookupSubjectsRequest {
+    ObjectType subject_type = 1;
+    string subject_relation = 2;
+    string relation = 3;
+    ObjectReference resource = 4;
+    optional uint32 limit = 5;
+    optional string continuation_token = 6;
+}
```

Comment:
Couple of things to consider but might be okay follow ups:

- There is some interesting stuff around wildcards with limits, but it might be something we follow up on afterwards. For example SpiceDB may return wildcards in addition to "concrete" relationships, so you'd get potentially limit + 1 results which a client might not expect, although this behavior can be elided with a wildcard option. Should we define to exclude wildcards so limit behavior is less surprising? 🤔 
- We may want to encapsulate limiting and/or continuation token into their own messages, in case related fields need to be added in the future.

Reply:
Went ahead and encapsulated the pagination inputs and outputs and exclude wildcard subjects until we figure out how we want to enable that.

Reply:
This is nice 👍